test/manual/atlas-data-lake-testing/atlas_data_lake_testing.prose.js

```diff
@@ -30,7 +31,7 @@ describe('Atlas Data Lake - prose', function () {
    * If a driver constructs and issues killCursors commands in other ways (e.g. public API), this test MUST be adapted to test all such code paths.
    */
   it('1. Test that the driver properly constructs and issues a killCursors command to Atlas Data Lake.', async function () {
-    client = new MongoClient('mongodb://mhuser:pencil@localhost');
+    client = new MongoClient('mongodb://mhuser:pencil@localhost', { family: 4 });
```

Comment:
Do you want to leave comments (maybe even with a `TODO` here) to switch this over to happy eyeballs once the driver supports a Node.js version that has it?